biogrid/libs/biogrid-simulator/src/lib/biogrid-state/biogrid-state.spec.ts

```diff
@@ -88,15 +100,15 @@ describe('classes', () => {
     expect(state.getAllVertices().length).toEqual(expectedGraph.nodes().length);
 
     // The expected object return from the shortest distances must be the same
-    expect(Object.keys(actualShortestDistances).length).toEqual(Object.keys(expectedShortestdistances).length);
+    expect(Object.keys(actualShortestDistances).length).toEqual(
+      Object.keys(expectedShortestdistances).length
+    );
     expect(actualShortestDistances).toEqual(expectedShortestdistances);
```

Comment:
I think this last expect statement is enough, and you don't have to explicitly compare lengths before